src/layouts/Docs.tsx

```diff
@@ -1,3 +1,288 @@
+const Content = (props: ChildOnlyProp) => {
+  const mdBreakpoint = useToken("breakpoints", "md")
+
+  return (
+    <Box
+      as="article"
+      flex={`1 1 ${mdBreakpoint}`}
+      maxW={{ base: "full", lg: mdBreakpoint }}
+      pt={{ base: 32, md: 12 }}
+      pb={{ base: 8, md: 16 }}
+      px={{ base: 8, md: 16 }}
+      m="0 auto"
+      sx={{
+        ".featured": {
+          paddingLeft: 4,
+          marginLeft: -4,
+          borderLeft: "1px dotted",
+          borderColor: "primary",
+        },
+        ".citation": {
+          p: {
+            color: "text200",
+          },
+        },
+      }}
+      {...props}
+    />
+  )
+}
+
+const BackToTop = (props: ChildOnlyProp) => (
+  <Flex
+    display={{ lg: "none" }}
+    mt={12}
+    pt={8}
+    borderTop="1px"
+    borderColor="border"
+    {...props}
+  >
+    <Link to="#top">
+      <Translation id="back-to-top" /> ↑
+    </Link>
+  </Flex>
+)
+
+export const docsComponents = {
+  a: MdLink,
+  h1: H1,
+  h2: H2,
+  h3: H3,
+  h4: H4,
+  p: Paragraph,
+  ul: UnorderedList,
+  ol: OrderedList,
+  li: ListItem,
+  pre: Codeblock,
+  ...mdxTableComponents,
+  Badge,
+  ButtonLink,
+  CallToContribute,
+  Card,
+  DeveloperDocsLinks,
+  Divider,
+  Emoji,
+  GlossaryTooltip,
+  InfoBanner,
+  RollupProductDevDoc,
+  SectionNav,
+  YouTube,
+}
+
+interface DocsLayoutProps extends MdPageContent, ChildOnlyProp {
+  frontmatter: DocsFrontmatter
+}
+
+export const DocsLayout = ({ children, frontmatter, slug, tocItems }: DocsLayoutProps) => {
+  const isRightToLeft = isLangRightToLeft(frontmatter.lang as Lang)
+  const isPageIncomplete = !!frontmatter.incomplete
+  const { asPath: relativePath} = useRouter()
+  const absoluteEditPath = `${EDIT_CONTENT_URL}${relativePath}`
+
+  return (
+    <Page dir={isRightToLeft ? "rtl" : "ltr"}>
+      {/* // TODO: IMPLEMENT PAGEMETADATA */}
+      {/* <PageMetadata
+        title={frontmatter.title}
+        description={frontmatter.description}
+      /> */}
+      <SideNavMobile path={relativePath} />
+      {isPageIncomplete && (
+        <BannerNotification shouldShow={isPageIncomplete}>
+          <Translation id="page-developers-docs:banner-page-incomplete" />
+        </BannerNotification>
+      )}
+      <ContentContainer>
+        <SideNav path={relativePath} />
+        <Content>
+          <H1 id="top">{frontmatter.title}</H1>
+          {frontmatter.lang !== "en" ? (
+            // TODO: Implement file contributors
+            <Text>CrowdinContributors</Text>
+            // <CrowdinContributors
+            //   relativePath={relativePath}
+            //   editPath={absoluteEditPath}
+            //   langContributors={allCombinedTranslatorsJson.nodes}
+            // />
+          ) : (
+            // TODO: Implement file contributors
+            // <GitHubContributors
+            //   relativePath={relativePath}
+            //   editPath={absoluteEditPath}
+            // />
+            <Text>GitHubContributors</Text>
```

Comment:
This component is already implemented in the tutorials, if we update the branch we should be able to implement it here as well. Not a blocker, we can do it in a separate PR.

Reply:
Brought this in!